functional_tests/imports/01-batch1-requirements.txt

```diff
@@ -0,0 +1,12 @@
+allennlp
+autogluon
+autokeras
+# avalanche
```

Comment:
why not just add these (and those commented out below) in now?

Reply:
what isnt tested shouldn't exist :)
If they are added, we might report them improperly, then we have tainted the analytics pipeline